.circleci/config.yml

```diff
@@ -58,6 +58,18 @@ executors:
       NPM_CONFIG_PREFIX: ~/.npm-global
     resource_class: large
 
+  heavy-test-executor:
+    parameters:
+      nodeversion:
+        type: string
+        default: *default_nodeversion
+    docker:
+      - image: circleci/node:<< parameters.nodeversion >>-browsers
+    working_directory: ~/ng
+    environment:
+      NPM_CONFIG_PREFIX: ~/.npm-global
+    resource_class: 2xlarge+
```

Comment:
This PR adds a new job  called `integration` that takes a total of ~57 minutes to run and replaces the current `test-large` and `test-large-ve` jobs. The last `test-large` on master took ~43 minutes total, and `test-large-ve` took 22m. These ran in parallel though, and on a less powerful machine, so it's not an obvious improvement.

I used a `2xlarge+` machine because on https://github.com/angular/angular/pull/33927/files#diff-1d37e48f9ceff6d8030570cd36286a61R297 I saw one being used as well, but I think there's room for exploring improvements.

Things that I should test:
- less powerful machine
- more parallelism at bazel level
- more parallelism at CI job level

Reply:
Using `large` executor instead of `2xlarge+` increased the job time from 43 minutes to 63 minutes, ~50% increase in time. But according to https://circleci.com/pricing/, `large` is 20 credits per minute and `2xlarge+` is 100. A 50% increase in time sounds reasonable for a 80% decrease in price.